src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Constants.cs

```diff
@@ -61,5 +61,28 @@ internal class Constants
                                                             Name = Constants.SPDXName,
                                                             Version = Constants.SPDXVersion
                                                         };
+
+        internal const int ReadBufferSize = 30;
+        internal const int MaxReadBufferSize = 32768;
```

Comment:
Why these values?

Reply:
Oh good catch, these were for testing, I changed it back to 4096.

This size should be greater than the largest token you can have in your JSON file. 4096 is a good buffer size in general and should accommodate all the parameters that we have